FFMpegCore/FFMpegCore.csproj

```diff
@@ -38,6 +38,7 @@
 
   <ItemGroup>
     <PackageReference Include="Instances" Version="2.0.0" />
+    <PackageReference Include="PInvoke.Kernel32" Version="0.7.104" />
```

Comment:
It looks like [`PInvoke.Kernel32`](https://www.nuget.org/packages/PInvoke.Kernel32/) is Windows-only

![image](https://user-images.githubusercontent.com/11181960/215896526-e0519fff-97e9-4f95-a4ee-582c084602ca.png)

Reply:
I have rebased, so CI can verify if cross-platform

Reply:
![image](https://user-images.githubusercontent.com/11181960/215897310-cc258fb8-5b20-4708-96b1-0cf8defa26a5.png)

Reply:
So although I see the value of this, on Windows, this library is for cross-platform use so I'll remove this